ReSharper.FSharp/src/FSharp.Psi.Daemon/src/QuickDoc/FSharpQuickDocProvider.fs

```diff
@@ -24,9 +24,16 @@ module FSharpQuickDoc =
 
         // todo: fix getting qualifiers
         let tokenNames = [token.Name]
+        let treeEndOffset =
+            if (isNull token.Parent) then token.GetTreeEndOffset() else
```

Comment:
Didn't the first Parent already have some active pattern-like type?
Maybe worth checking on that as well.